src/pkg/projects/framework.packaging.targets

```diff
@@ -3,6 +3,266 @@
+
+      <JsonControlProperty Include="priority" String="standard" />
+      <JsonControlProperty Include="section" String="libs" />
+      <JsonControlProperty Include="architecture" String="amd64" />
+
+      <JsonLicenseProperty Include="type" String="MIT and ASL 2.0 and BSD" />
+      <JsonLicenseProperty Include="full_text" String="$(FullLicenseText)" />
+    </ItemGroup>
+
+    <GenerateJsonObjectString Properties="@(JsonReleaseProperty)">
+      <Output TaskParameter="Json" PropertyName="JsonReleaseObject" />
+    </GenerateJsonObjectString>
+    <GenerateJsonObjectString Properties="@(JsonControlProperty)">
+      <Output TaskParameter="Json" PropertyName="JsonControlObject" />
+    </GenerateJsonObjectString>
+    <GenerateJsonObjectString Properties="@(JsonLicenseProperty)">
+      <Output TaskParameter="Json" PropertyName="JsonLicenseObject" />
+    </GenerateJsonObjectString>
+
+    <ItemGroup>
+      <CommonJsonProperty Include="package_name" String="$(VersionedInstallerName)" />
+      <CommonJsonProperty Include="short_description" String="$(MSBuildProjectName) $(InstallerPackageVersion)" />
+      <CommonJsonProperty Include="maintainer_name" String=".NET Core Team" />
+      <CommonJsonProperty Include="maintainer_email" String="dotnetpackages@dotnetfoundation.org" />
+      <CommonJsonProperty Include="install_root" String="/usr/share/dotnet" />
+      <CommonJsonProperty Include="long_description" String=".NET Core is a development platform that you can use to build command-line applications, microservices and modern websites. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/dotnet/core). We happily accept issues and PRs." />
+      <CommonJsonProperty Include="homepage" String="https://github.com/dotnet/core" />
+      <CommonJsonProperty Include="copyright" String="2017 Microsoft" />
+      <CommonJsonProperty Include="release" Object="$(JsonReleaseObject)" />
+      <CommonJsonProperty Include="control" Object="$(JsonControlObject)" />
+      <CommonJsonProperty Include="license" Object="$(JsonLicenseObject)" />
+    </ItemGroup>
+  </Target>
+
+  <!--
+    Copy the files in the package's data/ dir to a layout directory. This is what the targeting pack
+    installer will place in the dotnet install dir.
+
+    This extracts from the nupkg (zip) directly. An alternative would be using the PackageFile
+    items, but there's some risk of handling TargetPath metadata and filtering symbols differently
+    than the NuGet tasks do. Using the nupkg directly ensures results are identical.
+
+    This is not ideal: the dependency on NuGet package creation is avoidable. We could create (a)
+    layout(s) that both NuGet packaging and installers are based on.
+  -->
+  <Target Name="CreateFrameworkPackLayout"
```

Comment:
This still depends on the nupkg, I just moved the target. (@ericstj)

I experimented with separating out a layout step, but I didn't get to the point where it was actually used to create the nupkg so haven't included that. I still think that's the right way to go--something I realized too is that I think it might make it easier to transition to the built-in SDK Pack targets.